frontend/web/components/ProjectUsage.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC, useEffect, useState } from 'react'
+import Project from 'common/project'
+import InfoMessage from './InfoMessage'
+import { useGetProjectQuery } from 'common/services/useProject'
+import { useGetEnvironmentsQuery } from 'common/services/useEnvironment'
+import { Environment } from 'common/types/responses'
+const _data = require('common/data/base/_data')
+
+type ProjectUsageType = {
+  projectId: string
+}
+
+const ProjectUsage: FC<ProjectUsageType> = ({ projectId }) => {
+  const { data: project } = useGetProjectQuery({ id: projectId })
+  const { data: environments } = useGetEnvironmentsQuery({ projectId })
+
+  const [segmentOverridesUsage, setSegmentOverridesUsage] = useState<number>(0)
+
+  const getSegmentOverridesUsage = (apiKeys: string[]) => {
+    apiKeys.forEach((apiKey) => {
+      _data
+        .get(`${Project.api}environments/${apiKey}`)
+        .then((res: Environment) => {
+          setSegmentOverridesUsage(
+            (prev) => prev + (res.total_segment_overrides || 0),
+          )
+        })
+    })
+  }
+
+  useEffect(() => {
+    if (environments) {
+      const apiKeys = environments.results.map((env) => env.api_key)
+      getSegmentOverridesUsage(apiKeys)
+    }
+  }, [environments])
+
+  return (
+    <div className='mt-4'>
+      <Flex>
+        <Row>
+          <InfoMessage>
+            In order to ensure consistent performance, Flagsmith has some{' '}
+            <a
```

Comment:
In order to ensure consistent performance, Flagsmith has the following usage limits. <a ...>Check the Docs for more details</a>

Reply:
Done!